pkg/api/http/directmessaging_test.go

```diff
@@ -727,6 +728,36 @@ func TestV1DirectMessagingEndpoints(t *testing.T) {
 		assert.Equal(t, "ERR_DIRECT_INVOKE", resp.ErrorBody["errorCode"])
 	})
 
```

Comment:
I think you can test the `NotReady` error like this:

```suggestion
t.Run("Invoke returns error - no direct messaging ", func(t *testing.T) {
		fakeServer := newFakeHTTPServer()
		testAPI := &api{
			directMessaging: nil,
			universal: universal.New(universal.Options{
				CompStore:  compStore,
				Resiliency: resiliency.New(nil),
			}),
		}
		fakeServer.StartServer(testAPI.constructDirectMessagingEndpoints(), nil)

		apiPath := "v1.0/invoke/noDirectMessaging/method/fakeMethod"
		fakeData := []byte("fakeData")

		mockDirectMessaging.Calls = nil // reset call count

		mockDirectMessaging.
			On(
				"Invoke",
				mock.MatchedBy(matchContextInterface),
				mock.MatchedBy(func(b string) bool {
					return b == "noDirectMessaging"
				}),
				mock.AnythingOfType("*v1.InvokeMethodRequest"),
			).
			Return(nil, errors.New("NO TARGET_APP_ID")).
			Once()

		resp := fakeServer.DoRequest("POST", apiPath, fakeData, nil)

		assert.Equal(t, 500, resp.StatusCode)
		assert.Equal(t, "ERR_DIRECT_INVOKE", resp.ErrorBody["errorCode"])
	})
```